app/assets/stylesheets/govuk_publishing_components/components/helpers/_govuk-frontend-settings.scss

```diff
@@ -13,3 +13,4 @@
 
 $govuk-image-url-function: "app-image-url";
 $govuk-font-url-function: "app-font-url";
+$govuk-new-typography-scale: true;
```

Comment:
Nitpick: it might help clarify some of these changes if the commit message included a link to the change in `govuk-frontend` that they refer to e.g. in this case https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md#weve-adjusted-our-responsive-type-scale